src/components/sidebar/SideBarHome.module.sass

```diff
@@ -198,7 +198,8 @@ $breakpoint-tablet: 500px
     .nav
         width: 0
         padding: 0
-        box-shadow: none
+        box-shadow: 1px 1px 2px 2px tint(grey,50%)
```

Comment:
Added this box shadow to differentiate it from the rest of the page when the sidebar is opened on small screens